### PR TITLE
chore: Update imgutil

### PR DESCRIPTION
**Pull request by jabrown85**
Signed-off-by: Jesse Brown <jabrown85@gmail.com>

### CODE REVIEW

File: tools/go.sum [hunk 1]
```diff
@@ -134,11 +135,15 @@ github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V
 github.com/go-logr/logr v0.1.0/go.mod h1:ixOQHD9gLJUVQQ2ZOR7zLEifBX6tGkNJF4QyIY7sIas=
 github.com/go-logr/logr v0.2.0/go.mod h1:z6/tIYblkpsD+a4lm/fGIIU9mZ+XfAiaFtq7xTgseGU=
 github.com/go-ole/go-ole v1.2.1/go.mod h1:7FAglXiTm7HKlQRDeOQ6ZNUHidzCWXuZWq/1dTyBNF8=
+github.com/go-openapi/jsonpointer v0.0.0-20160704185906-46af16f9f7b1/go.mod h1:+35s3my2LFTysnkMfxsJBAMHj/DoqoB9knIWoYG/Vk0=
```
Comment (yaelharel) [inline tools/go.sum:138]: Do you understand why it was added?
I don't see it as part of:
https://github.com/buildpacks/imgutil/commit/cc867729ddde7cefe3c08e3b714e34856a7cc67b.

File: tools/go.sum [hunk 2]
```diff
@@ -134,11 +135,15 @@ github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V
...
 github.com/go-openapi/jsonpointer v0.19.2/go.mod h1:3akKfEdA7DF1sugOqz1dVQHBcuDBPKZGEoHC/NkiQRg=
 github.com/go-openapi/jsonpointer v0.19.3/go.mod h1:Pl9vOtqEWErmShwVjC8pYs9cog34VGT37dQOVbmoatg=
+github.com/go-openapi/jsonreference v0.0.0-20160704190145-13c6e3589ad9/go.mod h1:W3Z9FmVs9qj+KR4zFKmDPGiLdk1D9Rlm7cyMvf57TTg=
```
Comment (yaelharel) [inline tools/go.sum:141]: Same question as https://github.com/buildpacks/lifecycle/pull/539/files#r588573048

File: tools/go.sum [hunk 3]
```diff
@@ -134,11 +135,15 @@ github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V
...
 github.com/go-openapi/jsonreference v0.19.2/go.mod h1:jMjeRr2HHw6nAVajTXJ4eiUwohSTlpa0o73RUL1owJc=
 github.com/go-openapi/jsonreference v0.19.3/go.mod h1:rjx6GuL8TTa9VaixXglHmQmIL98+wF9xc8zWvFonSJ8=
+github.com/go-openapi/spec v0.0.0-20160808142527-6aced65f8501/go.mod h1:J8+jY1nAiCcj+friV/PDoE1/3eeccG9LYBs0tYvLOWc=
```
Comment (yaelharel) [inline tools/go.sum:144]: Same question as https://github.com/buildpacks/lifecycle/pull/539/files#r588573048

File: tools/go.sum [hunk 4]
```diff
@@ -134,11 +135,15 @@ github.com/go-logfmt/logfmt v0.4.0/go.mod h1:3RMwSq7FuexP4Kalkev3ejPJsZTpXXBr9+V
...
 github.com/go-openapi/spec v0.19.3/go.mod h1:FpwSN1ksY1eteniUU7X0N/BgJ7a4WvBFVA8Lj9mJglo=
+github.com/go-openapi/swag v0.0.0-20160704191624-1d0bd113de87/go.mod h1:DXUve3Dpr1UfpPtxFw+EFuQ41HhCWZfha5jSVRG7C7I=
```
Comment (yaelharel) [inline tools/go.sum:146]: Same question as https://github.com/buildpacks/lifecycle/pull/539/files#r588573048

**Review by yaelharel — APPROVED** [review]
There are a few additions that have been added automatically (I guess) and I'm not sure why, but other than that. Looks good to me.

File: tools/go.sum (hunk 1 above)
Comment (jabrown85) [reply tools/go.sum:138]: My answer for this is: go tooling is confusing.

It comes from ggcr. I think the latest ggcr update brought it in, but was `go mod tidy` wasn't run in the `tools/*` module during that PR.

**Review by ekcasey — APPROVED** [review]

**Pull request merged**